### ISSUE TITLE
Everyone thinks the big Plus is a button

### ISSUE THREAD

**Issue by qodfathr** [fabric8io/fabric8-ui#605]
This happens all over the dsaas, and even in other Patternfly-based products. The big "circle plus" icon

![image](https://cloud.githubusercontent.com/assets/16322190/24875888/a13dbbee-1df7-11e7-9a20-48e59f29b2e9.png)

sure does seem like a button. People keep clicking on it, expecting it to do something. It consumes a ton of screen real-estate, yet is non-functional. Worse, the real buttons are often (1) far away and (2) much smaller. Net result, you often do not even notice the real buttons,

**Comment by alexeykazakov** [comment]
+100! I keep clicking on that + :)

**Comment by joshuawilson** [comment]
@catrobson can you comment on this?

**Comment by ldimaggi** [comment]
+1 Agreed - it looks like a non-functioning button.

**Comment by catrobson** [comment]
@agalante will take a look at this and provide a suggestion on improvement.

**Comment by andresgalante** [comment]
Hi, you can make the `div` an anchor `a` and modify the styles a bit to look as it should:

First the HTML, I’ll add a prefix but you can overwrite pf class if you want.
```
<a class="blank-slate-pf alm-blank-slate-pf>
```
Instead of a `<div>`

Then write a bit of CSS to make it block, change the colors and add a hover:
```
.alm-blank-slate-pf {
display: block;
color: black;
text-decoration: none
cursor: pointer;
}

.alm-blank-slate-pf:hover {
box-shadow: 0 1px 6px rgba(3,3,3,.175);
}
```

Let me know if this helps!

**Comment by qodfathr** [comment]
What exacerbates the problem is that in other parts of the UI, there are blue circle-pluses, and those are buttons to add something.

**Comment by andresgalante** [comment]
Sorry, I can't really help with that. That seems like a customization you've made adding anchors around the icons. So I guess you'll have to chose one or the other.

**Comment by catrobson** [comment]
The blue icons are standalone icons in the UI instead of "add" buttons everywhere.  I think Andres's fix will make this area clickable now so I think this resolves our problem.

**Comment by mindreeper2420** [comment]
@catrobson I can fix this, if desired. It would go in under the Planner repo.

**Comment by joshuawilson** [comment]
That would be great, thanks @mindreeper2420

**Comment by mindreeper2420** [comment]
Here is the updated Blank State:
Static:
<img width="1466" alt="screen shot 2017-04-10 at 8 58 03 pm" src="https://cloud.githubusercontent.com/assets/4032718/24888313/89fec9bc-1e30-11e7-995d-4f8355380f20.png">

Hover:
<img width="1455" alt="screen shot 2017-04-10 at 8 57 44 pm" src="https://cloud.githubusercontent.com/assets/4032718/24888318/8de4da08-1e30-11e7-861f-c2ad1a438b5c.png">

@joshuawilson I'm unable to have the `onClick` activate the Work Item Type selector for adding a new Work Item. Can you take a look? The calls appear to be there, but they are not firing (and no errors).

**Comment by qodfathr** [comment]
I don't feel that the text "You can add one by clicking here, or by either of the + icons" is necessary. It also creates an unnecessary dependency between that text and the UI (e.g., if the UI changes, we'd need to remember to update this text.)

**Comment by andresgalante** [comment]
@mindreeper2420 Sorry, color should be `#363636` and not `black;`
and remove the blue on hover with the same color declaration.

Cards don't have a color change on hover and this should be consistent.

**Comment by mindreeper2420** [comment]
This is what a user will see when they are logged in and no Work Items are present, or none match the filtered criteria:
<img width="709" alt="screen shot 2017-04-14 at 9 46 55 am" src="https://cloud.githubusercontent.com/assets/4032718/25044909/6383d308-20f7-11e7-88bf-907b1e778617.png">

This is what a user will see when they are logged out and no Work Items match the filtered criteria:
<img width="691" alt="screen shot 2017-04-14 at 9 47 14 am" src="https://cloud.githubusercontent.com/assets/4032718/25044922/8241d024-20f7-11e7-86b8-00e951ca64de.png">

**Comment by joshuawilson** [comment]
Why an "i"? Why not an "L" or a closed pad lock? 
If we go with this, then we might want to add this to the existing pattern that @dlabrecq has done.
Have we answered the question that most of the devs have, namely this looks like a button and we have other "plus" buttons used for adding but this is not a button.

**Comment by mindreeper2420** [comment]
@joshuawilson I went with the 'i' instead of a lock as the lock made it feel 'bad' in some way. The 'i' gave off (to me) the idea that the user has done nothing wrong, and can continue if they log in (and, in essence, create an account with us).

When a user is logged in, this will be a clickable element. When the user hovers over the blank state, it will 'pop' out at them and the cursor will change to the hand (as all of our clickable elements should). You can check out this branch to see it in action: https://github.com/mindreeper2420/fabric8-planner/tree/blank-state-update

**Comment by michaelkleinhenz** [comment]
I tried to aggregate all above info into a PR. See here: https://github.com/fabric8io/fabric8-planner/pull/1685

**Comment by michaelkleinhenz** [comment]
Fixed with #1685.
